### PR TITLE
created mobile opt out

### PR DESCRIPTION
**Pull request by allisonsparrow**
for members of our mobile onboarding, we'd like to create an easy way for them to opt out of our onboarding without navigating our subscription center. By visiting this page they will stop receiving mobile onboarding emails, while still remain subscribed to other email subscriptions.

### CODE REVIEW

File: website/opt-out-mo/index.hbs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+compact_header: true
+---
+  <div class="container">
+    <div class="row">
+      <div class="columns twelve">
+        <h1><center>You have successfully opted out of Optimizely for iOS Onboarding</center></h1>
+        <p>Other resources you may be interested in:</p>
+          <ul>
+            <li>Optiverse</li>
+            <li>Knowledge Base</li>
```
Comment (kylerush) [inline website/opt-out-mo/index.hbs]: @allisonsparrow these should be links, right now they aren't.

just change them to:

`
<li><a href="https://community.optimizely.com">Optiverse</a></li>
<li><a href="http://help.optimizely.com">Knowledge Base</a></li>
`

after that we're all set to deploy.

**Comment by dtothefp** [comment]
@allisonsparrow can you the comments above that @kylerush made so we can merge?

**Comment by stewsmith** [comment]
@allisonsparrow What is the status of this PR? I can finish the changes that @kylerush proposed, or we can abandon the issue. The PR should not stay unresolved though.

**Comment by stewsmith** [comment]
@allisonsparrow please lmk about the above comment.

**Pull request closed without merging**